src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js

```diff
@@ -90,7 +90,8 @@ function resetComponentState(component) {
 class CustomActiveDonut extends PureComponent {
   constructor(props) {
     super(props);
-    this.state = { activeIndex: 0 };
+    const { data } = props;
+    this.state = { activeIndex: data.length - 1 };
```

Comment:
Can you add a default value if data.length come as undefined? So it will not break in future

Reply:
updated